exporter/newrelicexporter/transformer.go

```diff
@@ -34,6 +34,8 @@ const (
 	collectorVersionKey       = "collector.version"
 	instrumentationNameKey    = "instrumentation.name"
 	instrumentationVersionKey = "instrumentation.version"
+	droppedAttributesCountKey = "otel.dropped_attributes_count"
+	droppedEventsCountKey     = "otel.dropped_events_count"
```

Comment:
Maybe this?
otel.event.dropped_attributes_count

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/jaeger.md#events

Reply:
I updated the events to use `otel.event.dropped_attributes_count` to follow the precedent in that spec, but I kept these names for dropped counts on spans and logs.